server/ocfg/config.go

```diff
@@ -0,0 +1,163 @@
+package ocfg
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+)
+
+var (
+	AllowOrigins      []string
```

Comment:
This is a super nit but we could add comments to these with the equivalent env var for gopls to highlight